eoscostarica/eden-accounting-standard
# Fix/webapp issues

### Fix/webapp issues

### What does this PR do?

- Resolve #171 
- Resolve #172 
- Resolve #173  
- Resolve #174 

### Steps to test

1. Run project
2. Must see this:
![imagen](https://user-images.githubusercontent.com/49013556/197617368-40ea1774-e472-44d0-ac77-5d06ebac76c2.png)
![imagen](https://user-images.githubusercontent.com/49013556/197617505-69e6caa3-b3a0-4677-a4bd-9931fb23c091.png)


#### CheckList
